src/main/java/mate/academy/springboot/swagger/util/SortingOrders.java

```diff
@@ -0,0 +1,29 @@
+package mate.academy.springboot.swagger.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.data.domain.Sort;
+
+public class SortingOrders {
+    public static List<Sort.Order> getOrders(String sortBy) {
```

Comment:
let's also return `Sort` object and make this method non-static, so `SortingOrders` will be Component